src/collections/_documentation/development/sdk-dev/unified-api.md

```diff
@@ -54,7 +54,20 @@ meant that certain integrations (such as breadcrumbs) were often not possible.
 
 - **integration**: Code that provides middlewares, bindings or hooks into certain frameworks or environments, along with code that inserts those bindings and activates them. Usage for integrations does not follow a common interface.
 
-- **event processors:** Are callbacks that run for every event. They can either return a "new" event which in most cases means just adding data OR return `null` in case the event will be dropped and not sent.
+- **event processors**: Callbacks that run for every event.  
+  They can either return a "new" event, which in most cases means just adding
```

Comment:
suggest using italics for _new_ here rather than quotes.

Can we make this more clear? The phrase "which in most cases means just adding data" suggests that other cases exist. 

Suggest "These either return a _new_ event or `null`. New events typically add data. `Null` is discarded and does not process further."

Reply:
I actually changed this completely, as the *new* was a bit misleading.
You return the event that you get after modifications (move semantics).
While it is technically possible to create a *completely new* event, I don’t think that is a valid usecase actually.